src/ansys/fluent/core/workflow.py

```diff
@@ -18,6 +18,13 @@
 from ansys.fluent.core.utils.fluent_version import FluentVersion
 
 
+class CommandInstanceCreationError(Exception):
```

Comment:
Is it better to derive the custom exception from more specific exception class, `RuntimeError` in this case? If user wants to handle execptions in a general manner, they'll need to catch generic `Exception` in this case which is not a good practice.

Reply:
I agree that in this case inheriting from `RuntimeError` would work fine and be potentially more useful in some use cases, I am going to include that as part of the changes here. Thanks for the suggestion.